Util/Config.py

```diff
@@ -6,7 +6,7 @@
 fetch = True
 since = "4 years ago"
 verbose = 0
-sections = ["Hyper-V CORE AND DRIVERS"]
+sections = ["Hyper-V/Azure CORE AND DRIVERS"]
```

Comment:
```suggestion
sections = ["Hyper-V CORE AND DRIVERS", "Hyper-V/Azure CORE AND DRIVERS"]
```

I would suggest having both the old and new names to cover the case where downstream distros don't immediately update the maintainers file. But I haven't checked and that may not actually be a concern.